### PR TITLE
Disable flaky `test_poa_multiple_producers` test

### PR DESCRIPTION
**Pull request by rafal-ch**
This PR disables the flaky `test_poa_multiple_producers` test.

Follow-up issue: https://github.com/FuelLabs/fuel-core/issues/2351

### Before requesting review
- [X] I have reviewed the code myself
- [X] I have created follow-up issues caused by this PR and linked them here

### CODE REVIEW

**Review by xgreenx — DISMISSED** [review]

**Review by AurelienFT — DISMISSED** [review]

**Comment by xgreenx** [comment]
You can also remove the assert that causes the error below.

image

**Review by xgreenx — APPROVED** [review]

**Review by AurelienFT — APPROVED** [review]

File: crates/services/src/sync_processor.rs [hunk 1]
```diff
@@ -209,6 +209,8 @@ mod tests {
         // Then
         while broadcast_receiver.recv().await.is_ok() {}
         assert!(instant.elapsed() >= Duration::from_secs(10));
+        // Wait for the metrics to be updated.
+        tokio::time::sleep(Duration::from_secs(1)).await;
```
Comment (rymnc) [inline crates/services/src/sync_processor.rs:213]: is this change intended

**Comment by rafal-ch** [comment]
> You can also remove the assert that causes the error below.
> image

I see you merged, but what about this comment? Is this no longer needed due to your changes to `crates/services/src/sync_processor.rs`?

**Comment by xgreenx** [comment]
Yeah, I think my change should fix flakyness
